### PR TITLE
83 navigability in the UI

### PR DESCRIPTION
**Pull request by Mister-Mario**
Testing sonar cloud code coverage

### CODE REVIEW

File: gatewayservice/gateway-service.js [hunk 1]
```diff
@@ -58,6 +58,36 @@
...
+  }
+});
+
+
+app.get('/questions/:lang/:amount', async (req, res) => {
+  try {
+    const lang = req.params.lang;
+    const amount = req.params.amount;
+    // Forward the question request to the quetion service
+    const questionResponse = await axios.get(questionServiceUrl+'/questions/' + lang + '/' + amount);
```
[the remark this answers is not in the document]
Comment (Mister-Mario) [reply gatewayservice/gateway-service.js:82]: Fixed

File: gatewayservice/gateway-service.js [hunk 2]
```diff
@@ -58,6 +58,36 @@
   }
 });
 
+app.get('/questions/:lang/:amount/:type', async (req, res) => {
+  try {
+    const lang = req.params.lang;
+    const amount = req.params.amount;
+    const type = req.params.type;
+    // Forward the question request to the quetion service
+    const questionResponse = await axios.get(questionServiceUrl+'/questions/' + lang + '/' + amount + '/' + type);
```
[the remark this answers is not in the document]
Comment (Mister-Mario) [reply gatewayservice/gateway-service.js:67]: Fixed

File: gatewayservice/gateway-service.js [hunk 3]
```diff
@@ -81,6 +111,27 @@
...
+  } catch (error) {
+    res.send(error);
+  }
+});
+
+app.get('/record/ranking/:user', async(req, res)=>{
+  try {
+    const user = req.params.user;
+    // Forward the record request to the record service
+    const recordResponse = await axios.get(recordServiceUrl + '/record/ranking/' + user);
```
[the remark this answers is not in the document]
Comment (Mister-Mario) [reply gatewayservice/gateway-service.js:128]: Fixed

**Comment by lauratbg** [comment]
Nice

**Pull request merged**